modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -96,15 +96,13 @@ This command performs a number of tests on the provided files to test if they ar
 . Restart services to pick up the changes:
 +
 ----
-mgrctl exec -- spacewalk-service stop
-mgrctl exec -- systemctl restart postgresql.service
-mgrctl exec -- spacewalk-service start
+mgradm restart
 ----
 
 
 If you are using a proxy, you need to generate a server certificate RPM for each proxy, using their host names and cnames.
-You should use [command]``mgr-ssl-cert-setup`` also on a {productname} Proxy to replace the certificates.
-Because the {productname} Proxy does not have a {postgresql} database, only [command]``spacewalk-service restart`` is sufficient.
+You should use [command]``mgr-ssl-cert-setup`` also on a {productname} Proxy container host to replace the certificates.
+On {productname} Proxy,[command]``mgrpxy restart``.
```

Comment:
Also the proxies have containers and the certificates needs to be replaced in some containers. I am not sure how to do it. I think you need to replace the config.
@cbosdo @mbussolotto - please provide instructions on how to replace certificates on a proxy.

Reply:
I added some such adjustments.  Of course, I'd appreciate more improvement proposals. @cbosdo @mbussolotto , please see above.

Reply:
`mgr-ssl-setup` won't help for proxies. The easiest solution for proxies is to generate a new configuration tarball and deploy it. In fact you can still just generate the certificate and copy it in the proper config file... but it's surely a bit complex for users.

Reply:
ok, replaced.